### PR TITLE
For vendors: Header section

### LINKED ISSUE #495: [vendor page] Add new header image and link to contact us
The goal of this header is to provide an overview of the "story" of MobiMart for a vendor and to provide a clear CTA to contact us, via a button.

Screen Shot 2023-11-28 at 4 54 40 PM

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #495 

## What this PR does
- Adds new content section (consisting of an SVG with an Alt tag, a title, paragraphs and a button)
- This change uses ALL the same HTML/CSS as what the home page top section has.
- Does this PR change any CSS? No.

### CODE REVIEW

File: src/for-vendors.html [hunk 1]
```diff
@@ -3,3 +3,37 @@
...
 ---
+
+<div class="container">
+  <div class="imaged-section imaged-section--right">
+    <div class="imaged-section__title">
+      <h1 class="home-hero-title">Join us in bringing industry standards to California’s transit providers</h1>
+    </div>
+    <div class="imaged-section__image">
+      <img
+        alt="A woman at a stand with a sign that reads, 'Next stop: Better products'"
```
Comment (machikoyasuda) [inline src/for-vendors.html:14]: @mrose914 @segacy1 I need a better alt text tag for this image. It's a woman....standing at a .......... lemonade stand (???????) with bubbles that have the refresh logo (????)

**Comment by machikoyasuda** [comment]
There are some spacing issues here (like between paragraphs, buttons, and container alignment details), but would like to leave it to this #490 issue for a full update.

Example: I _COULD_ rewrite ALL paragraphs to have the same margin/padding-bottom, but I'd need confirmation from @segacy1 that this change is okay for ALL paragraphs across ALL the pages. Or I could write really hacky CSS that does this: The current style says that there should be `.7rem` under each paragraph, which totals `11.2px`. Then I have to subtract: `24px-11.2px` to come up with what remainder padding I should add in addition to the .7, to make this page look like what is on Figma. Messy.  cc @thekaveman @angela-tran

**Comment by thekaveman** [comment]
> but would like to leave it to this https://github.com/cal-itp/mobility-marketplace/issues/490 issue for a full update.

Agree! Let's worry about all that at once.

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran The code is ready to review. Otherwise, just awaiting a better alt image tag line.

**Comment by segacy1** [comment]
@machikoyasuda the png is a bit blurry, could we try it as an SVG? 
vendor-hero-lg
vendor-hero-sm

**Comment by segacy1** [comment]
@machikoyasuda for alt tag, how's this: A saleswoman behind a stand for "Better Products," framed by refresh and checkmark icon illustrations.

102 characters

**Comment by machikoyasuda** [comment]
With the inclusion of the alt tag for the illustration, this PR is now ready for review @thekaveman @angela-tran

**Comment by angela-tran** [comment]
@machikoyasuda What do you think about using the SVG that Segacy uploaded in https://github.com/cal-itp/mobility-marketplace/pull/504#issuecomment-1841131072? The PNG is blurry for me too

**Comment by machikoyasuda** [comment]
@angela-tran @segacy1 

The SVG is much less blurry - way better than the PNG: 
image

**Review by angela-tran — APPROVED** [review]

**Pull request merged**